extensions/azurecore/src/azureResource/interfaces.ts

```diff
@@ -42,4 +42,24 @@ export interface IAzureResourceTenantService {
 export interface IAzureResourceNodeWithProviderId {
 	resourceProviderId: string;
 	resourceNode: azureResource.IAzureResourceNode;
-}
\ No newline at end of file
+}
+
+export interface AzureSqlResource {
+	name: string;
+	loginName: string;
+}
+
+export interface IAzureResourceService<T extends AzureSqlResource> {
```

Comment:
>IAzureResourceService [](start = 17, length = 21)

So are you following some convention here for interfaces that start with an i or not? Not really a huge deal but it would be nice if we came to some sort of consensus on this at some point (then again vs code isn't so maybe not worth trying to figure something out)

Reply:
It seemed to be this way, I prefer it the other way since it doesn't seem to be a typescript pattern but trying not to break the patter in this extension, at least.